### ISSUE TITLE
Updating Filter Reverts to GGG Regular Filter

### ISSUE THREAD

**Issue by FirePenguinMaster** [notablackbear/poe_qolV2#64]
I can get the filter to run correctly initially, but hitting "Refresh" on the Overlay window and then refreshing the Item Filter regresses entirely to the standard GGG filter, apparently with no custom rules. Is this a bug? Am I doing something wrong?

**Comment by Khaze413** [comment]
I have the same problem. I have tried everything I can think of and just when I think I get it working it reverts to the base game filter. I checked the POEQOL_Base.filter AFTER this had reverted and it looks like the entire "Chaos Recipe" section is gone.

**Comment by DrLeh** [comment]
Having the same issue, it looks like it's just replacing the entirety of the base filter with the `chaos_items_filter.filter` file

**Comment by danybellefeuille** [comment]
Yeah refreshing the filter resets it to default GGG filter... any solution?

**Comment by RemiBorgen** [comment]
I had the same problem and found out that when i customized the filter in filterblade, the start/end section for chaos recipe went missing. See this comment in the setup "#### Right now it depends on finding 234hn50987sd to know when the chaos recipe section starts and looks for 2345ina8dsf7 to know what line it stops at. 5head, I know."

If that is missing, just open the filter that came with the zip and copy paste that section into the filter. I dunno if the placement matters, but i did it at the end of the unique section.

Edit:Placement matters but not sure where it's optimal. I've placed it in the 6s and 5l section for now
